tests/e2e-tests/config/setup.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+const {
+	setupSettings,
+	createTaxes,
+	createCoupons,
+	createProducts,
+	createShippingMethods,
+	enablePaymentGateways,
+} = require( './fixture-loaders' );
+const { setup: setupPuppeteer } = require( 'jest-environment-puppeteer' );
+
+module.exports = async ( globalConfig ) => {
+	await setupPuppeteer( globalConfig );
+	Promise.all( [
+		setupSettings(),
+		createTaxes(),
+		createCoupons(),
+		createProducts(),
+		createShippingMethods(),
+		enablePaymentGateways(),
+	] ).then( ( results ) => {
+		const [ , taxes, coupons, products, shippingMethods ] = results;
+		global.wc = {
```

Comment:
`global` is preserved and can be used inside tests and in the teardown phase.

Reply:
Not sure on the naming of this, could be clearer - `global` is very generic and `wc` is pretty generic too :)

Is this something like `testDataIds` ?

Reply:
Ah, just realised this is node `global`.

Reply:
TIL,  I assumed it was provided by Jest and not node, but yeah `global` is something we don't control. wc is just a placeholder, there is no risk of this data touching anything else, this global in particular (this execution context) is not shared with tests or other things.

Reply:
Just to be clear, the name `wc` is still a bit too generic in my opinion - I'd prefer it has a more specific, clearer name. For example `fixtureData` or `testDataIds` (`= { products: array of ids, coupons:  etc }`). Not a blocker.